app/src/main/java/org/shadowice/flocke/andotp/View/EntriesCardAdapter.java

```diff
@@ -539,7 +540,7 @@ public void editEntryTags(final int pos) {
             @Override
             public Object call() throws Exception {
                 entries.get(realPos).setTags(tagsAdapter.getActiveTags());
-                DatabaseHelper.saveDatabase(context, entries, encryptionKey);
+                saveEntries(false);
```

Comment:
Id set this to true, to create a backup when we change tags

Reply:
If we do that, then we also have to set it to true after editing the label, issuer and thumbnail. I wouldn't mind that, but it could result in a lot of backups again. What do you think?

Reply:
So my thought for this feature was to allow someone to pretty much restore and go from where they left off, rather than just providing a backup of the latest codes only.

I'm not 100% sure I know how HOTP codes work, but if we don't backup on each code change, would that not mean people have to update loads of times on a restore to get it to work again?

Maybe for now we leave it set to false, and down the road we can do another iteration that adds a setting to chose from a basic/full auto backups? Or change it so that if you don't have the append date setting set it just overrides the same file (so it doesn't create a new file each time)

Reply:
Sounds good to me. So we will leave it like this for now and iterate on it in future versions. Maybe we just add a multi-select preference that allows the user to specify additional triggers for the auto-backup function. And overwriting the backup file in case the "append date" option is disabled sounds good as well.